HB/common/log.elpi

```diff
@@ -420,30 +421,31 @@ coq.vernac->ppinductive (inductive ID IsInd Arity Ks) ParamsRev PP :-
     coq.mk-app x ParamsAsArgs (X x),
     coq.vernac->ppinductiveconstructor (Ks (X x)) KsPp.
 
-pred coq.vernac->ppinductiveconstructor i:list indc-decl, o:list coq.pp.
+func coq.vernac->ppinductiveconstructor list indc-decl -> list coq.pp.
 coq.vernac->ppinductiveconstructor [] [].
 coq.vernac->ppinductiveconstructor [constructor ID Arity|Ks] PP :-
   PP = [str ID,{coq.arity->pp Arity},SEP|Rest],
   if (Ks = []) (SEP = str"") (SEP = glue [brk 1 0, str "| "]),
   coq.vernac->ppinductiveconstructor Ks Rest.
 
-pred coq.vernac->ppinductiveparams i:list (pair implicit_kind term), o:list coq.pp.
+func coq.vernac->ppinductiveparams list (pair implicit_kind term) -> list coq.pp.
 coq.vernac->ppinductiveparams [] [].
 coq.vernac->ppinductiveparams [pr Imp T|Rest] PP :-
   PP = [box (hov 2) [str A,ID,str " : ", TY,str B]|PPRest],
-  coq.term->pp T ID, decl T _ Ty, coq.term->pp Ty TY,
+  coq.term->pp T ID, std.once(decl T _ Ty), coq.term->pp Ty TY,
```

Comment:
Decl is definitely a function (from coq elpi)

Reply:
Yep, I will change that.